dot/network/proto/api.v1.proto

```diff
@@ -29,8 +29,6 @@ message BlockRequest {
 		// Start with given block number.
 		bytes number = 3;
 	}
-	// End at this block. An implementation defined maximum is used when unspecified.
-	bytes to_block = 4; // optional
```

Comment:
Should we reserve the field so index `4` is not used in the future? As in that could have bad consequences. I'm not sure if this is needed, maybe not if we don't care about compatibility.

https://developers.google.com/protocol-buffers/docs/proto3#reserved

Reply:
In the substrate PR, they just remove the field from the message https://github.com/paritytech/substrate/pull/12447/files#diff-f2c2fea2718de075610d4e9425fcf161ddef69d0c7a23a3d2ace3810123c5dd1

But I can use the `reserved` keyword